### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by evelinabart** [JMSaindon/NlpSquad#1]
I'm getting an error in the evaluation part:  "'str' object has no attribute 'detach'". 
It's related with 
def to_list(tensor):
    return tensor.detach().cpu().tolist()

and 

output = [to_list(output[i]) for output in outputs]
